packages/rainbowkit/src/utils/locale.ts

```diff
@@ -26,7 +26,50 @@ export const locales = [
   'ru-RU',
 ];
 
-// Check if browser locale is supported
-export const isBrowserLocaleSupported = (locale: string) => {
-  return locales.includes(locale as Locale);
+export const isIncluded = (keys: string[], key: string) => {
+  return keys.includes(key);
+};
+
+export const standardizeBrowserLocale = (
+  locale: string,
+): Locale | undefined => {
+  if (isIncluded(['en-US', 'en'], locale)) {
+    return 'en-US';
+  }
+
+  if (isIncluded(['es', 'es-419'], locale)) {
+    return 'es-419';
+  }
+
+  if (isIncluded(['fr', 'fr-FR'], locale)) {
+    return 'fr-FR';
+  }
+
+  if (isIncluded(['pt', 'pt-BR'], locale)) {
+    return 'pt-BR';
+  }
+
+  if (isIncluded(['pt', 'pt-BR'], locale)) {
+    return 'pt-BR';
+  }
+
+  if (isIncluded(['zh', 'zh-CN'], locale)) {
+    return 'zh-CN';
+  }
+
+  if (isIncluded(['id', 'id-ID'], locale)) {
+    return 'id-ID';
+  }
+
+  if (isIncluded(['hi', 'hi-IN'], locale)) {
+    return 'hi-IN';
+  }
+
+  if (isIncluded(['tr', 'tr-TR'], locale)) {
+    return 'tr-TR';
+  }
+
+  if (isIncluded(['ru', 'ru-RU'], locale)) {
+    return 'ru-RU';
+  }
 };
```

Comment:
A bit of a cleaner way to implement this:
```ts
export const standardizeBrowserLocale = (
  locale: string,
): Locale | undefined => ({
  'ru-RU': 'ru-RU',
  'ru': 'ru-RU', 
})[locale] as Locale;
```

Reply:
Nice! Yeah object structuring s the best way to go instead of repeating the same function over and over again 👍

https://github.com/rainbow-me/rainbowkit/pull/1534/commits/241a70739ce3a07223e327382b8ab7ff686b4420#diff-97daf67abd2e298c4f7264deb2b51e364f392924ad5f82a793ff38dfb75544daR33-R53